setup.py

```diff
@@ -6,19 +6,20 @@
 
```

Comment:
That is a useful feature.  It might work in other versions, so didn't want to be overly restrictive.   I guess we could do something like 2.7, >=3.3, < 4.

@ludwigschwardt You might have an opinion on this, with having to support various versions of Python in things like katpoint and katdal?

Reply:
If it runs on 3.4 / 3.5, it might as well be allowed. We still have 16.04 servers with Python 2.7 + 3.5. Our sort-of standard approach (taken from katpoint) is

```Python
python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, <4'
```
[I forgot why `>=3.3` does not work]

Reply:
Thanks @ludwigschwardt.  I have done so.